public/stylesheets/app.css

```diff
@@ -155,3 +155,12 @@ body.dark-mode main strong,
 body.dark-mode main a {
 	color: var(--dark-mode-fg-color);
 }
+
+.control.is-required::before {
```

Comment:
Getting closer! I think the asterisk should be put after the label. It's currently ending up before the whole field due to it being added to the `div.control` element.
![localhost_1337_nmap (2)](https://github.com/ronin-rb/ronin-app/assets/12671/c78e07a9-e9fe-4bbc-9f70-f0ba66d3cc46)